### PR TITLE
Known Issues Pages Issue #130

### PR DESCRIPTION
**Pull request by samirmansour**
https://osc.github.io/ood-documentation-test/known-issues/issues/overview.html

Added the implementation of the known issues pages. Still working on the format of the pages.

### CODE REVIEW

File: source/issues/dashboard.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+=========
+.. note::
+	 Last Update: 10:36 AM
+
+Duplicate Sub Apps
+------------------
+
+Duplicate sub apps may appear if ``/etc/ood/config/apps/$APP`` contains a **mainfest.yml** file
+
++------------+------------+-----------+
```
Comment (MorganRodgers) [inline source/issues/dashboard.rst]: Looks like a TODO

File: source/issues/shell.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+As you can see, when the text was pasted it was pasted twice.
+
+.. note::
+	 If you would like to track the progress of this issue you can go on `Github <https://github.com/OSC/ood-shell/issues/59/>`_
+
+Issue Tracking
+---------------------------
+
++------------+---------------------+
```
Comment (MorganRodgers) [inline source/issues/shell.rst]: Another TODO

File: source/issues/overview.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+| File names with       |                       |
+| X?HTML are not fully  |  `Link <FileNames_>`_ |
+| supported.            |                       |
++-----------------------+-----------------------+
+| Uploading large files |                       |
+| is not reliable.      |  `Link <LargeFile_>`_ |
+|                       |                       |
++-----------------------+-----------------------+
+| Attempting to download|                       |
+| forbidden directory   |  `Link <DoloForb_>`_  |
```
Comment (MorganRodgers) [inline source/issues/overview.rst]: I had good chuckle at Doloforb.

File: source/issues/overview.rst [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
++=======================+=======================+
+| Invalid               |                       |
+| OOD_LOCALES_ROOT      |  `Link <NoMethod_>`_  |
+| causes NoMethodError. |                       |
++-----------------------+-----------------------+
+| Processing erb creates|                       |
+| performance issues.   |  `Link <ERBPer_>`_    |
+|                       |                       |
++-----------------------+-----------------------+
+| Batch connect adpater |                       |
```
Comment (MorganRodgers) [inline source/issues/overview.rst]: Speeling errar in adpater.

File: source/issues/overview.rst [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+| errors do not contain |  `Link <ERBPer_>`_    |
+| stack trace.          |                       |
++-----------------------+-----------------------+
+| Processing erb creates|                       |
```
Comment (MorganRodgers) [inline source/issues/overview.rst]: This is a duplicate and has the Link for BatchEr.

**Comment by MorganRodgers** [comment]
@ericfranz do you feel that the tables are sufficient or do we want a little more exposition on the issues?

**Comment by ericfranz** [comment]
The tables should use the list table format which is much easier to maintain than the current table layout. For example https://osc.github.io/ood-documentation/master/customization.html#branding table is produced by this code:

```rst
.. list-table:: Branding
   :header-rows: 1
   :stub-columns: 1

   * - Feature
     - Environment Variable
     - Details
   * - Title
     - OOD_DASHBOARD_TITLE
     - The title appears in the navbar and is controlled by the environment variable ``$OOD_DASHBOARD_TITLE``. The default value is "Open OnDemand".
   * - Logo
     - OOD_DASHBOARD_LOGO
     - The default value for ``OOD_DASHBOARD_LOGO`` is ``/public/logo.png`` and this should be the URL to the logo. By default if you place a logo.png at ``/var/www/ood/public/logo.png`` it will be accessible via the URL ``https://your.ondemand.institution.edu/public/logo.png``.
   * - Favicon
     - OOD_PUBLIC_URL
     - The favicon is expected to exist at the path ``$OOD_PUBLIC_URL/favicon.ico``. For a default OOD installation the favicon will be located at ``/var/www/ood/public/favicon.ico``.
   * - Brand background color
     - OOD_BRAND_BG_COLOR
     - Controls the background color of the navbar in the dashboard
   * - Brand foreground color
     - OOD_BRAND_LINK_ACTIVE_BG_COLOR
     - Controls the background color the active link in the navbar in the dashboard
   * - Replace header title with logo
     - OOD_DASHBOARD_HEADER_IMG_LOGO
     - Value should be url to logo i.e. ``/public/logo.png``.  the background color the active link in the navbar in the dashboard
   * - Use white text on black background for navbar.
     - OOD_NAVBAR_TYPE
     - By default we use ``inverse`` for this value, which specifies to use `Bootstrap 3's inverted navbar <https://getbootstrap.com/docs/3.3/components/#navbar-inverted>`_ where text is white and background is black (or dark grey). You can set this to ``default`` to use black text on light grey background if it fits your branding better.
```

**Comment by ericfranz** [comment]
Though was actually thinking that we would have a page that looked more like https://osc.github.io/ood-documentation/master/customization.html

In the sense that we had a clear list and we might consider for some of the known issues to explain workarounds or show a screenshot that provides context that an end user or admin might appreciate, but could be considered noise to a developer on the GitHub issue. Lots of the GitHub issues lack context or workarounds for users because as developers we have that context.

**Comment by ericfranz** [comment]
Lots of thoughts! Lets discuss next week.

**Pull request merged**